### ISSUE TITLE
Current default css for error toasts doesn't meet WCAG 1.4.3

### ISSUE THREAD

**Issue by gavinbarron** [fkhadra/react-toastify#477]
This is a bug report.

The default CSS for `.Toastify__toast--error` provides insufficent contrast ratio with the default `#ffffff` text.

Until this we have had no CSS overrides.

I'm using the Accessiblity Insights extension to scan quickly for accessibility issues and happened to run a scan with an error toast open and got the following results

Path: .Toastify__toast-body
```
<div role="alert" class="Toastify__toast-body">There was an issue retrieving your profile information. Please refresh the page and retry or contact an administrator if the problem persists.</div>
```
Fix the following:Element has insufficient color contrast of 3.82 (foreground color: #ffffff, background color: #e74c3c, font size: 11.3pt (15px), font weight: normal). Expected contrast ratio of 4.5:1

Path: button
```
<button class="Toastify__close-button Toastify__close-button--error" type="button" aria-label="close">✖︎</button>
```
Fix the following:Element has insufficient color contrast of 2.57 (foreground color: #f8c9c5, background color: #e74c3c, font size: 10.5pt (14px), font weight: bold). Expected contrast ratio of 4.5:1

**Comment by fkhadra** [comment]
Hey @gavinbarron,

Thank you for reporting that. Could you share which extension you use, it's really interesting.

**Comment by gavinbarron** [comment]
We're using the [Accessibility Insights](https://accessibilityinsights.io/) extension which uses axe-core under the hood.
[Chrome Store](https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni)
[EdgeDev store](https://microsoftedge.microsoft.com/addons/category/EdgeExtensionsEditorsPick)

**Comment by fkhadra** [comment]
@gavinbarron thanks to you I found that error is not the only one concerned. I also found other quirks. I'll work on them for the next release.

**Comment by gavinbarron** [comment]
Thank you! What do you mind sharing what else you found?

**Comment by fkhadra** [comment]
Sure. BTW, I didn't use the extension, there is an accessibility tool built-in with firefox.

- The contrast for success and warning is not sufficient
- Clickable elements must be focusable and should have interactive semantics
- Focusable element may be missing focus styling

**Comment by fkhadra** [comment]
After havind a look at different design system/css framework (bootstrap, uber, ibm etc...) None of them seems to fully meet WCAG. I'll close this issue has this can be fixed by the end user. 
But I'll still had a mention in the documention for that.
